docs/customize/toc.md

```diff
@@ -164,6 +164,7 @@ your introduction page entry (the first entry in `_toc.yml`). For example:
 This will cause all chapters of the book to be
 numbered. They will follow a hierarchy according to the sub-sections structure
 defined in your `_toc.yml` file.
+You can also limit the TOC numbering depth by setting the `numbered` flag to an integer instead of `true`, e.g., `numbered: 3`.
```

Comment:
```suggestion
You can also **limit the TOC numbering depth** by setting the `numbered` flag to an integer instead of `true`, e.g., `numbered: 3`.
```